core/new-gui/src/app/common/type/user-file.ts

```diff
@@ -4,7 +4,8 @@
  * Corresponds to `/web/src/main/java/edu/uci/ics/texera/web/resource/UserFileResource.java`
  */
 export interface UserFile extends Readonly<{
-  id: number;
+  uid: number;
```

Comment:
regarding the comment above: "Corresponds to `/web/src/main/java/edu/uci/ics/texera/web/resource/UserFileResource.java`". Can we update the comment and point to the new backend java/scala file, and possible the sql file? Also the backend should have a comment that points to here. We want all relevant type definitions to point to each other.

Reply:
added